operator/integration-tests/model-in-image/run-test.sh

```diff
@@ -54,6 +54,7 @@ function usage() {
     WORKDIR               : /tmp/\$USER/mii-sample-work-dir
     DOMAIN_NAMESPACE      : sample-domain1-ns
     MODEL_IMAGE_NAME      : model-in-image 
+    MODEL_IMAGE_TAG       : ${WDT_DOMAIN_TYPE}-v2
```

Comment:
Finally, none of the java IT tests are setting this variable. So how is this change to "run-test.sh" changing IT test behavior?

Reply:
The MODEL_IMAGE_TAG is set in ItMiiSampleHelper.java which is used by ItMiiSampleWlsAux.java, ItMiiSampleWlsMain.java, ItMiiSampleFmwAux.java and ItMiiSampleFmwMain.java. The reason for this change is that in the old way, the tests set image name as format mii-<timestamp>-<namespace>:wls-v1. When we push the image to phx.ocir.io, each image will create a new repository. When we hit the repository limit in phx.ocir.io, we can not push the image anymore.

Reply:
How about instead of replacing the entire generated tag, instead _prepend_ extra information to the tag? There's much descriptive value in the tag that's useful for debugging/tracing/image-tracking:  the generated tag indicates version (v1 or v2), whether or not image is aux, and whether image is WLS or JRF.  Furthermore, the generated tag is intended to differentiate the different use cases run by the test, so when you eliminate the differentiation, I assume the command line is no longer safe to run multiple use cases, since each use case will have the same tag (luckily the IT tests only run one use case at a time, so I assume that helps)...